test/scripts/apidoc/verify-jsdoc-tags.spec.ts

```diff
@@ -201,6 +201,22 @@ describe('verify JSDoc tags', () => {
                 expect(link, 'Expect method reference to contain ()').toContain(
                   ')'
                 );
+                expect(
+                  link,
+                  "Expect method reference to have a ': ' after the parenthesis"
+                ).toContain('): ');
+                expect(
+                  link,
+                  'Expect method reference to have a description starting with a capital letter'
+                ).toMatch(/\): [A-Z]/);
+                expect(
+                  link,
+                  'Expect method reference to have a description sentence'
+                ).toMatch(/\): .{10,}$/);
```

Comment:
Should we change it like this:

```suggestion
                expect(
                  link,
                  'Expect method reference to start with a standard description phrase'
                ).toMatch(
                  /\): (?:For generating |For more information about |For using |For the replacement method)/
                );
```